kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm-ui/src/build/revapi-config.json

```diff
@@ -26,7 +26,18 @@
   "ignores": {
     "revapi": {
     "_comment": "Changes between 7.48.0.Final and the current branch. These changes are desired and thus ignored.",
-          "ignore": []
+          "ignore": [
```

Comment:
@MarianMacik I want to you check this one, it's related to revapi. Do you have better options ?
directly change revapi definition config.

Reply:
That is fine, we want to keep changes in API.